docs/computing/connecting/index.md

```diff
@@ -0,0 +1,93 @@
+# Connecting to CSC supercomputers
+
+There are two main ways of connecting to CSC supercomputers. The traditional way to
+work on a supercomputer is [using an SSH client](#using-an-ssh-client). We
+also offer a [web interface](#using-the-web-interface) to our systems, which
+enables running both graphical applications and command-line shells.
```

Comment:
```suggestion
enables running both graphical applications and command-line shells, in your web browser without the need to set up anything.
```
Or something like this, saying it is really nice starting point?

Reply:
Thanks! Added in commit 08fe55c6

Reply:
I think we should be a bit careful here. I think for many users the web interface might also not be a good starting point -- it depends on what you want to accomplish. Since this is not a getting started page, maybe we should focus on maintaining an as neutral language as possible so as to address all users.

Reply:
Agree, could replacing "anything"  from the suggestion with "SSH-keys" do that? Then we do not say that it is a nice starting point, but still make everyone aware that you do not need to set them up when using the webinterface?

Reply:
Well, in principle you don't need SSH keys to use SSH client either (although it is recommended). I would honestly prefer to keep the text just as it was.

Reply:
Reverted in 3b4a752